packages/core/src/nativeModulesTypes.ts

```diff
@@ -0,0 +1,241 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+/**
+ * A configuration object to initialize Datadog's features.
+ */
+export class DdNativeSdkConfiguration {
+    constructor(
+        readonly clientToken: string,
+        readonly env: string,
+        readonly applicationId: string,
+        readonly nativeCrashReportEnabled: boolean,
+        readonly sampleRate: number,
+        readonly site: string,
+        readonly trackingConsent: string,
+        readonly additionalConfig: object
+    ) {}
+}
```

Comment:
is this needed?

Reply:
If it is not used in the code base I guess you can remove it.

Reply:
Actually it's worth having it here, I like to have native types decorelated from JS types. That way we can easily update the native types without changing the API for RN users :)